windows/main.c

```diff
@@ -1225,6 +1226,15 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PSTR cmd, int n
                         debug("Please specify correct theme (please check user manual for list of correct values).");
                         theme = THEME_DEFAULT;
                     }
+                    argv_theme=1;
+                }
+            } else if(!wcscmp(arglist[i], L"--profile")) {
```

Comment:
can you support `--profile=savename.tox` in addititon to `--profile savename.tox` or provide a reason you don't want to?